ui/src/app/workflows/pagination.ts

```diff
@@ -0,0 +1,7 @@
+export interface WorkflowsPagination {
```

Comment:
I imagine this could just live next `WorkflowsPaginationOptions` in `models/workflows.ts`, no?

Reply:
Oh. that is more natural! I'll reflect this opinion. Thank you :)